specification/threatmodel.adoc

```diff
@@ -57,7 +57,7 @@ in-scope adversaries
 T6: Incorrect execution of workload via runtime modification of CPU 
 registers, CSRs, mode switches via in-scope adversaries
 
-T7: Invalid code execution or data injection/replacement via second stagel 
+T7: Invalid code execution or data injection/replacement via second stage
```

Comment:
should we call this G-stage?

Reply:
Both `G-stage` (5 time) and `second stage` (17 times) are used through out the spec. If we change this I would suggest we change all instances of this to `G-stage`. Personally I find `second stage` less confusing. `G-stage` sounds like OS stage translation (1st stage) but then I see the term `VS-stage` only then `G-stage` makes sense.

Reply:
The priv spec mentions G-stage and KVM code also revised sometime back to follow the G-stage convention instead of 2nd stage. It would be good to align with those.

Reply:
Sounds good. Let me then move to `G-stage`.